### ISSUE TITLE
new feature proposal: enabling tent respawn (Zeus backpacks)

### ISSUE THREAD

**Issue by mike-187** [LordGolias/antistasi#147]
A proposal for v0.3:

Would it be difficult to integrate an option so the commander or group leader can carry a backpack (i.e. "B_Respawn_TentA_F") which he can build a mobile respawn tent? (this backpacks have been introduced with the Zeus update and I believe one has been added with the Malden update)

Not sure how it can be integrated into the mission as the revive and respawn is not the standard BI script/module. So basically the Players can choose to respawn at HQ or the tent.

few suggested to have this Zeus option available as it enables the other players to re-join the fight quicker if they bleed out or respawn right away at the HQ.

If the vanilla modules would break the mission... there might be other scripts available as ie. the Wasteland mission also uses some custom tent respawn point script.

**Comment by Warhammer47** [comment]
Nice idea!
Although interesting what is the difference between this and the Camp establishment option (where you can fast travel to)?
In my personal opinion I think that with an HQ away from the theater of operations (action) makes the game harder and more realistic

**Comment by mike-187** [comment]
Oh did not try the camps yet... are u able to fast travel there even when enemy is nearby?

Also the respawn at the tent would be a empty (no gear) respawn for the players... so also more risky.

**Comment by Warhammer47** [comment]
True, with enemies nearby you cannot fast travel in camps (both your location and the camps location)
